[INFO ][20180110 17:29:26,253][AbstractApplicationContext:589] Refreshing org.springframework.context.annotation.AnnotationConfigApplicationContext@5577140b: startup date [Wed Jan 10 17:29:26 CST 2018]; root of context hierarchy
[INFO ][20180110 17:29:26,390][XmlBeanDefinitionReader:316] Loading XML bean definitions from class path resource [rabbitmq.properties]
[WARN ][20180110 17:29:26,477][AbstractApplicationContext:557] Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.xml.XmlBeanDefinitionStoreException: Line 1 in XML document from class path resource [rabbitmq.properties] is invalid; nested exception is org.xml.sax.SAXParseException; lineNumber: 1; columnNumber: 1; 前言中不允许有内容。
[INFO ][20180110 17:30:33,480][AbstractApplicationContext:589] Refreshing org.springframework.context.annotation.AnnotationConfigApplicationContext@5577140b: startup date [Wed Jan 10 17:30:33 CST 2018]; root of context hierarchy
[INFO ][20180110 17:30:33,625][XmlBeanDefinitionReader:316] Loading XML bean definitions from class path resource [rabbitmq.properties]
[WARN ][20180110 17:30:33,687][AbstractApplicationContext:557] Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.xml.XmlBeanDefinitionStoreException: Line 1 in XML document from class path resource [rabbitmq.properties] is invalid; nested exception is org.xml.sax.SAXParseException; lineNumber: 1; columnNumber: 1; 前言中不允许有内容。
[INFO ][20180110 17:31:16,254][AbstractApplicationContext:589] Refreshing org.springframework.context.annotation.AnnotationConfigApplicationContext@5577140b: startup date [Wed Jan 10 17:31:16 CST 2018]; root of context hierarchy
[INFO ][20180110 17:31:16,383][XmlBeanDefinitionReader:316] Loading XML bean definitions from class path resource [rabbitmq.properties]
[WARN ][20180110 17:31:16,433][AbstractApplicationContext:557] Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.xml.XmlBeanDefinitionStoreException: Line 1 in XML document from class path resource [rabbitmq.properties] is invalid; nested exception is org.xml.sax.SAXParseException; lineNumber: 1; columnNumber: 1; 前言中不允许有内容。
[INFO ][20180110 17:32:37,880][AbstractApplicationContext:589] Refreshing org.springframework.context.annotation.AnnotationConfigApplicationContext@5577140b: startup date [Wed Jan 10 17:32:37 CST 2018]; root of context hierarchy
[INFO ][20180110 17:32:37,980][XmlBeanDefinitionReader:316] Loading XML bean definitions from class path resource [rabbitmq.properties]
[WARN ][20180110 17:32:38,033][AbstractApplicationContext:557] Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.xml.XmlBeanDefinitionStoreException: Line 1 in XML document from class path resource [rabbitmq.properties] is invalid; nested exception is org.xml.sax.SAXParseException; lineNumber: 1; columnNumber: 1; 前言中不允许有内容。
[INFO ][20180110 17:33:59,522][AbstractApplicationContext:589] Refreshing org.springframework.context.annotation.AnnotationConfigApplicationContext@5577140b: startup date [Wed Jan 10 17:33:59 CST 2018]; root of context hierarchy
[INFO ][20180110 17:33:59,650][XmlBeanDefinitionReader:316] Loading XML bean definitions from class path resource [rabbitmq.properties]
[WARN ][20180110 17:33:59,652][AbstractApplicationContext:557] Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanDefinitionStoreException: IOException parsing XML document from class path resource [rabbitmq.properties]; nested exception is java.io.FileNotFoundException: class path resource [rabbitmq.properties] cannot be opened because it does not exist
[INFO ][20180110 17:34:17,981][AbstractApplicationContext:589] Refreshing org.springframework.context.annotation.AnnotationConfigApplicationContext@5577140b: startup date [Wed Jan 10 17:34:17 CST 2018]; root of context hierarchy
[INFO ][20180110 17:34:18,075][XmlBeanDefinitionReader:316] Loading XML bean definitions from class path resource [rabbitmq.properties]
[WARN ][20180110 17:34:18,122][AbstractApplicationContext:557] Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.xml.XmlBeanDefinitionStoreException: Line 1 in XML document from class path resource [rabbitmq.properties] is invalid; nested exception is org.xml.sax.SAXParseException; lineNumber: 1; columnNumber: 1; 前言中不允许有内容。
[INFO ][20180110 17:34:35,619][AbstractApplicationContext:589] Refreshing org.springframework.context.annotation.AnnotationConfigApplicationContext@5577140b: startup date [Wed Jan 10 17:34:35 CST 2018]; root of context hierarchy
[INFO ][20180110 17:34:35,734][XmlBeanDefinitionReader:316] Loading XML bean definitions from class path resource [rabbit.properties]
[WARN ][20180110 17:34:35,783][AbstractApplicationContext:557] Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.xml.XmlBeanDefinitionStoreException: Line 1 in XML document from class path resource [rabbit.properties] is invalid; nested exception is org.xml.sax.SAXParseException; lineNumber: 1; columnNumber: 1; 前言中不允许有内容。
[INFO ][20180110 17:35:26,240][AbstractApplicationContext:589] Refreshing org.springframework.context.annotation.AnnotationConfigApplicationContext@5577140b: startup date [Wed Jan 10 17:35:26 CST 2018]; root of context hierarchy
[INFO ][20180110 17:35:26,341][XmlBeanDefinitionReader:316] Loading XML bean definitions from class path resource [rabbitmq.properties]
[WARN ][20180110 17:35:26,388][AbstractApplicationContext:557] Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.xml.XmlBeanDefinitionStoreException: Line 1 in XML document from class path resource [rabbitmq.properties] is invalid; nested exception is org.xml.sax.SAXParseException; lineNumber: 1; columnNumber: 1; 前言中不允许有内容。
[INFO ][20180110 17:39:36,029][AbstractApplicationContext:589] Refreshing org.springframework.context.annotation.AnnotationConfigApplicationContext@5577140b: startup date [Wed Jan 10 17:39:36 CST 2018]; root of context hierarchy
[INFO ][20180110 17:39:36,143][XmlBeanDefinitionReader:316] Loading XML bean definitions from class path resource [rabbitmq.properties]
[WARN ][20180110 17:39:36,190][AbstractApplicationContext:557] Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.xml.XmlBeanDefinitionStoreException: Line 1 in XML document from class path resource [rabbitmq.properties] is invalid; nested exception is org.xml.sax.SAXParseException; lineNumber: 1; columnNumber: 1; 前言中不允许有内容。
[INFO ][20180110 17:41:18,318][AbstractApplicationContext:589] Refreshing org.springframework.context.annotation.AnnotationConfigApplicationContext@5577140b: startup date [Wed Jan 10 17:41:18 CST 2018]; root of context hierarchy
[WARN ][20180110 17:41:18,557][AbstractApplicationContext:557] Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'rabbitConfig': Unsatisfied dependency expressed through field 'channelCacheSize'; nested exception is org.springframework.beans.TypeMismatchException: Failed to convert value of type 'java.lang.String' to required type 'int'; nested exception is java.lang.NumberFormatException: For input string: "${channelCacheSize}"
[INFO ][20180110 17:42:25,014][AbstractApplicationContext:589] Refreshing org.springframework.context.annotation.AnnotationConfigApplicationContext@5577140b: startup date [Wed Jan 10 17:42:25 CST 2018]; root of context hierarchy
[WARN ][20180110 17:42:25,338][AbstractApplicationContext:557] Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'connectionFactory' defined in com.kevin.spring.rabbit.config.RabbitConfig: Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.springframework.amqp.rabbit.connection.ConnectionFactory]: Factory method 'connectionFactory' threw exception; nested exception is java.lang.NumberFormatException: For input string: "${channelCacheSize}"
[INFO ][20180110 17:46:53,991][AbstractApplicationContext:589] Refreshing org.springframework.context.annotation.AnnotationConfigApplicationContext@5577140b: startup date [Wed Jan 10 17:46:53 CST 2018]; root of context hierarchy
[WARN ][20180110 17:46:54,385][AbstractApplicationContext:557] Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'connectionFactory' defined in com.kevin.spring.rabbit.config.RabbitConfig: Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.springframework.amqp.rabbit.connection.ConnectionFactory]: Factory method 'connectionFactory' threw exception; nested exception is java.lang.NumberFormatException: For input string: "${channelCacheSize}"
[INFO ][20180110 17:47:26,862][AbstractApplicationContext:589] Refreshing org.springframework.context.annotation.AnnotationConfigApplicationContext@5577140b: startup date [Wed Jan 10 17:47:26 CST 2018]; root of context hierarchy
[INFO ][20180110 17:47:27,345][AbstractConnectionFactory:392] Created new connection: connectionFactory#10e41621:0/SimpleConnection@5b12b668 [delegate=amqp://kevin@127.0.0.1:5672/, localPort= 3353]
[INFO ][20180110 17:47:33,567][AbstractApplicationContext:589] Refreshing org.springframework.context.annotation.AnnotationConfigApplicationContext@5577140b: startup date [Wed Jan 10 17:47:33 CST 2018]; root of context hierarchy
[INFO ][20180110 17:47:34,243][AbstractConnectionFactory:392] Created new connection: connectionFactory#10e41621:0/SimpleConnection@206a70ef [delegate=amqp://kevin@127.0.0.1:5672/, localPort= 3365]
